tracer/test/Datadog.Trace.IntegrationTests/ContainerTaggingTests.cs

```diff
@@ -52,14 +41,21 @@ public async Task Http_Headers_Contain_ContainerId()
 
                 await tracer.FlushAsync();
 
-                var spans = agent.WaitForSpans(1);
-                Assert.Equal(1, spans.Count);
-                Assert.Equal(expectedContainedId, actualContainerId);
+                var spans = agent.WaitForSpans(count: 1);
+                Assert.Equal(expected: 1, spans.Count);
 
-                if (EnvironmentTools.IsWindows())
+                var headers = agent.TraceRequestHeaders.Should().ContainSingle().Subject;
```

Comment:
afaik `ContainSingle()` should always fail since there should be at least 6 standard headers in additional to the container id header, if present.
```suggestion
                var headers = agent.TraceRequestHeaders;
```

Reply:
Never mind.

"`TraceRequestHeaders` contains a collection of collections of headers"

Thanks for clarifying!